torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -925,7 +925,14 @@ struct CUDAFusedKernel : public FusedKernel {
     TORCH_NVRTC_CHECK(nvrtcGetPTXSize(program, &ptx_size));
     ptx.resize(ptx_size);
     TORCH_NVRTC_CHECK(nvrtcGetPTX(program, ptx.data()));
-
+//need an active context for cuModuleLoadData
```

Comment:
I'll be happy to merge that, but it will conflict with @mruberry's split, so you need to decide in which order do you want this to land.

Reply:
I'm happy to rebase after @mruberry's split lands.